standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -4664,6 +4664,58 @@ public void dropStoredProcedure(StoredProcedureRequest request) throws MetaExcep
     return client.get_all_stored_procedures(request);
   }
 
+  /**
+   * Builder for the GetProjectionsSpec. This is a projection specification for partitions returned from the HMS.
+   */
+  public static class GetPartitionProjectionsSpecBuilder {
+
+    java.util.List<java.lang.String> partitionList = null;
```

Comment:
Can we mark these fields as private? Any particular reason they are package private?

Reply:
I agree these should be private. I will change them.